extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [UNRELEASED]
 
-## [UNRELEASED]
+- Enable collection of telemetry for UI interactions in webviews. [#2114](https://github.com/github/vscode-codeql/pull/2114)
```

Comment:
Can you think of a different term to `webviews` ? I'm thinking that user's that might look at the CHANGELOG aren't necessarily familiar with the term.

Reply:
I could say "query results panel". Is that better? 🤔 

Or could say "UI interactions not linked to VS Code commands" since we already talk about telemetry of commands in the privacy document.

Reply:
Both ideas sounds good to me. Might be best to stick to the language that the privacy document is using.

Reply:
The relevant line from the privacy document is "Interactions with UI elements, including buttons, links, and other inputs" so I've gone with that.